aiida/cmdline/commands/cmd_node.py

```diff
@@ -27,15 +27,15 @@ def verdi_node():
 
 @verdi_node.group('repo')
 def verdi_node_repo():
-    pass
+    """Manage and inspect the content of a node repository folder."""
```

Comment:
There are no "managing" commands, only to inspect, correct?

Reply:
Correct.